src/features/settings/settingsSlice.ts

```diff
@@ -15,18 +14,19 @@ export interface SettingsState {
   explorerLiquidAPI: string;
   explorerLiquidUI: string;
   assets: Asset[];
+  proxyUrl: string;
+  isInitialized: boolean;
   marketsLabelled?: MarketLabelled[];
-  tdexDaemonBaseUrl: string;
-  macaroonCredentials?: string;
   tdexdConnectUrl?: string;
 }
 
 export const initialState: SettingsState = {
   chain: 'regtest',
   explorerLiquidAPI: 'https://blockstream.info/liquid/api',
   explorerLiquidUI: 'https://blockstream.info/liquid',
-  tdexDaemonBaseUrl: 'https://localhost:9000',
```

Comment:
if we want to keep the web version working (ie. as staging), we may want to be able to support to connect to daemon directly still. Should we just have a general `baseUrl` and via ENV based how we build/run we innject the right url with right protocol